index.js

```diff
@@ -30,6 +30,11 @@ module.exports = exports.default = function emailPropt({
   allowInvalidChars = false
 } = {}) {
   return new Promise((resolve, reject) => {
+    // some environments (e.g., cygwin) don't provide a tty
+    if (!process.stdin.setRawMode) {
```

Comment:
>  I don't intend to question your choice here

I appreciate the feedback, and take no offense. :)

> Do you have specific evidence for such a case?

I don't have evidence for a case where something isn't a tty and has `setRawMode` defined. I also have seen no evidence that it can't happen either. https://github.com/zeit/now-cli/issues/24 is the genesis of this fix, where `now-cli`is missing `setRawMode` in some environments. It seems safer to check specifically for the presence of that function, since that's the one in `email-prompt` that causes the error.